plone/app/event/profiles/default/actions.xml

```diff
@@ -27,7 +27,7 @@
    </property>
    <property name="visible">True</property>
   </object>
-  <object name="ical_import_disable" meta_type="CMF Action" i18n:domain="plone.app.event">
+  <object name="ical_import_disable" meta_type="CMF Action" i18n:domain="plone">
    <property name="title" i18n:translate="title_ical_import_disable">Disable icalendar import</property>
```

Comment:
Don't you need an upgrade step to change the domain of CMF Action?

Reply:
Good point, I guess so. I will add it later, thanks for the review!

Reply:
+1